<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="pt-br">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">139ae75371de424175cbd3094216b21901544bf5</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.Reflection.InvalidFilterCriteriaException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">796b86bc2a0c163ffc4a556b55582bf6d91f2505</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">f9d03e29118d282abbc58483031e069755255d26</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>The exception that is thrown in &lt;xref:System.Type.FindMembers*&gt; when the filter criteria is not valid for the type of filter you are using.</source>
          <target state="translated">A exceção que é lançada em &lt;xref:System.Type.FindMembers*&gt;quando os critérios de filtro não é válido para o tipo de filtro que você está usando.&lt;/xref:System.Type.FindMembers*&gt;</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT">
          <source><ph id="ph1">`InvalidFilterCriteriaException`</ph> uses the HRESULT COR_E_INVALIDFILTERCRITERIA which has the value 0x80131601.</source>
          <target state="translated"><ph id="ph1">`InvalidFilterCriteriaException`</ph>usa o COR_E_INVALIDFILTERCRITERIA HRESULT que tem o valor 0x80131601.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Reflection.InvalidFilterCriteriaException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with the default properties.</source>
          <target state="translated">Inicializa uma nova instância do <bpt id="p1">&lt;xref href="System.Reflection.InvalidFilterCriteriaException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> classe com as propriedades padrão.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor initializes an instance of <ph id="ph1">`InvalidFilterCriteriaException`</ph> with an empty message string and the root cause exception set to <ph id="ph2">`null`</ph>.</source>
          <target state="translated">Este construtor inicializa uma instância de <ph id="ph1">`InvalidFilterCriteriaException`</ph> com uma cadeia de caracteres de mensagem vazia e raiz causar exceção definida como <ph id="ph2">`null`</ph>.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor sets the properties of the <ph id="ph1">`Exception`</ph> object as follows:      |Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|null|   |&lt;xref:System.Exception.Message%2A&gt;|The empty string ("").|</source>
          <target state="translated">Este construtor define as propriedades da <ph id="ph1">`Exception`</ph> do objeto da seguinte maneira: | Propriedade | Valor |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| Nulo |   | &lt;xref:System.Exception.Message%2A&gt;| A cadeia de caracteres vazia (""). |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Reflection.InvalidFilterCriteriaException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with the given HRESULT and message string.</source>
          <target state="translated">Inicializa uma nova instância do <bpt id="p1">&lt;xref href="System.Reflection.InvalidFilterCriteriaException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> classe com a cadeia de caracteres HRESULT e a mensagem determinada.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.</source>
          <target state="translated">Uma exceção que é lançada como um resultado direto de uma exceção anterior deve incluir uma referência à exceção anterior no &lt;xref:System.Exception.InnerException%2A&gt;propriedade.&lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or <ph id="ph1">`null`</ph> if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.</source>
          <target state="translated">O &lt;xref:System.Exception.InnerException%2A&gt;propriedade retorna o mesmo valor que é passado para o construtor, ou <ph id="ph1">`null`</ph> se o &lt;xref:System.Exception.InnerException%2A&gt;propriedade não fornecer o valor da exceção interna para o construtor.&lt;/xref:System.Exception.InnerException%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.Reflection.InvalidFilterCriteriaException&gt;.</source>
          <target state="translated">A tabela a seguir mostra os valores de propriedade inicial para uma instância de &lt;xref:System.Reflection.InvalidFilterCriteriaException&gt;.&lt;/xref:System.Reflection.InvalidFilterCriteriaException&gt;</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">| Propriedade | Valor |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| <ph id="ph1">`null`</ph>|   | &lt;xref:System.Exception.Message%2A&gt;| A cadeia de caracteres de mensagem de erro. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve">
          <source>The message text for the exception.</source>
          <target state="translated">O texto da mensagem da exceção.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Reflection.InvalidFilterCriteriaException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with the specified serialization and context information.</source>
          <target state="translated">Inicializa uma nova instância do <bpt id="p1">&lt;xref href="System.Reflection.InvalidFilterCriteriaException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> classe com as informações de serialização e contexto especificadas.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source>A <bpt id="p1">&lt;xref uid="langword_csharp_SerializationInfo" name="SerializationInfo" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> object that contains the information required to serialize this instance.</source>
          <target state="translated">Um <bpt id="p1">&lt;xref uid="langword_csharp_SerializationInfo" name="SerializationInfo" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> objeto que contém as informações necessárias para serializar a essa instância.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>A <bpt id="p1">&lt;xref uid="langword_csharp_StreamingContext" name="StreamingContext" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> object that contains the source and destination of the serialized stream associated with this instance.</source>
          <target state="translated">Um <bpt id="p1">&lt;xref uid="langword_csharp_StreamingContext" name="StreamingContext" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> objeto que contém a origem e destino do fluxo serializado associado a essa instância.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.Reflection.InvalidFilterCriteriaException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Inicializa uma nova instância do <bpt id="p1">&lt;xref href="System.Reflection.InvalidFilterCriteriaException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> classe com uma mensagem de erro especificada e uma referência à exceção interna que é a causa dessa exceção.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.</source>
          <target state="translated">Uma exceção que é lançada como um resultado direto de uma exceção anterior deve incluir uma referência à exceção anterior no &lt;xref:System.Exception.InnerException%2A&gt;propriedade.&lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or <ph id="ph1">`null`</ph> if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.</source>
          <target state="translated">O &lt;xref:System.Exception.InnerException%2A&gt;propriedade retorna o mesmo valor que é passado para o construtor, ou <ph id="ph1">`null`</ph> se o &lt;xref:System.Exception.InnerException%2A&gt;propriedade não fornecer o valor da exceção interna para o construtor.&lt;/xref:System.Exception.InnerException%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.Reflection.InvalidFilterCriteriaException&gt;.</source>
          <target state="translated">A tabela a seguir mostra os valores de propriedade inicial para uma instância de &lt;xref:System.Reflection.InvalidFilterCriteriaException&gt;.&lt;/xref:System.Reflection.InvalidFilterCriteriaException&gt;</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|The inner exception reference.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">| Propriedade | Valor |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| A referência à exceção interna. |   | &lt;xref:System.Exception.Message%2A&gt;| A cadeia de caracteres de mensagem de erro. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">A mensagem de erro que explica o motivo da exceção.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">A exceção que é a causa da exceção atual.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve">
          <source>If the <ph id="ph1">`inner`</ph> parameter is not <bpt id="p1">&lt;xref uid="langword_csharp_null" name="null" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>, the current exception is raised in a <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept> block that handles the inner exception.</source>
          <target state="translated">Se o <ph id="ph1">`inner`</ph> parâmetro não é <bpt id="p1">&lt;xref uid="langword_csharp_null" name="null" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>, a exceção atual é gerada em um <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt> <ept id="p2">&lt;/xref&gt;</ept> bloco que manipula a exceção interna.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>